File: Arrays/C/1. Two_Sum.c
int* twoSum(int* nums, int numsSize, int target, int* returnSize) {
    *returnSize=2;
    int *returna=(int*)malloc(2*(sizeof(int)));
    for(int i=0;i<numsSize;i++) {
        for(int j=i+1;j<numsSize;j++) {
            int def[2] = {i,j};
            if(nums[i]+nums[j]==target)
                memcpy(returna, def, sizeof(def));
        }
    }
    return returna;
}

File: Arrays/Python/1. Two Sum.py
class Solution(object):
    def twoSum(self, nums, target):
        res = []
        for i, x in enumerate(nums):
            res.append([x, i])
        res.sort()
        
        left, right = 0, len(res) - 1
        while left < right:
            sum2 = res[left][0] + res[right][0]
            if sum2 == target:
                return [res[left][1], res[right][1]]
            elif sum2 > target:
                right -= 1
            else:
                left += 1
